src/test/java/co/cask/wrangler/steps/PipelineTest.java

```diff
@@ -93,4 +95,26 @@ public void testSplitWithNull() throws Exception {
     Assert.assertEquals("1,2,a,A", row.getValue("firstCol"));
     Assert.assertNull(row.getValue("secondCol"));
   }
+
+  @Test
+  public void testEscapedStrings() throws Exception {
+    List<Step> steps = new ArrayList<>();
+    Row row = new Row("__col", StringEscapeUtils.unescapeJava("1\\ta"));
```

Comment:
Should we escape once here and again at https://github.com/hydrator/wrangler-transform/pull/5/files#diff-01163bfca2584bef3a980fd885e184e1R101
Shouldn't we test with the replica of input as how it will be coming from FE